### PR TITLE
Uc 24(scrape all rate my prof pages)

### PR DESCRIPTION
**Pull request by seannam**

### CODE REVIEW

File: ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            }
+
+            return scrapeRMP;
+
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="ProfName"></param>
```
Comment (jltien) [inline ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs]: Maybe finish writing this header.

File: ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+            // Parse the JSON to get pk_id of the professor 
+            JObject responseObj = JObject.Parse(myContent);
+            JObject respondNode = (JObject)responseObj["response"];
+            JArray docsNode = (JArray)respondNode["docs"];
+            JValue pk_id = (JValue)docsNode[0]["pk_id"];
+
+            string id = pk_id.ToString();
+  
+            return (id);
```
Comment (harishduwadi) [inline ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs]: Should we just return this without using a variable? We could just return this as pk_id.ToString() as opposed to storing it in a variable.

File: ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            if (quality == null && wouldTakeAgain == null && difficultyLevel == null)
+            {
+                professorIsRated = false;
+            }
+
+            // Professor's Full Name
+            string professorFullName;
+            if (professorIsRated)
+            {
+                HtmlNode professorNode = htmlDoc.DocumentNode.SelectSingleNode(RateMyProfXPaths.ProfessorFirstNamePath);
```
Comment (seannam) [inline ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs]: Explain what this is doing.

File: ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+                "siteName=rmp&rows=20&group=off&group.field=content_type_s&group.limit=20&fq=content_type_s%3ATEACHER&fq=schoolname_t%3A%" +
+                "22University+of+California+San+Diego%22";
+
+            // Use urlString to extract the pk_id
+            string pk_id = (GetRequest(urlString)).Result;
+
+            // Return the final url containing pk_id
+            return ("http://www.ratemyprofessors.com/ShowRatings.jsp?tid=" + pk_id);
+        }
+
```
Comment (seannam) [inline ucsdscheduleme/Repo/ScrapeRateMyProfessor.cs]: Add method functions

**Comment by KennethLundberg** [comment]
These changes were moved to a different branch.

**Pull request closed without merging**